lib/images/controller_helper.rb

```diff
@@ -0,0 +1,12 @@
+module Images
+  module ControllerHelper
+    private
+
+    def image_by_id(image_id)
+      @image = Image.find(image_id)
```

Comment:
It's cool that you are extracting this to a module, but I am not sure if using instance variables in a module is good practice because if you include this module without knowing what is in it you might get problems, for example, variables are getting overridden.
I would recommend simply return the image here, and assign it in the controller

Reply:
I agree.